tests/TestCase/Model/Filter/DateBetweenTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+	namespace Search\Test\TestCase\Model\Filter;
+
+	use Cake\Core\Configure;
+	use Cake\I18n\Time;
+	use Cake\ORM\TableRegistry;
+	use Cake\TestSuite\TestCase;
+	use Search\Manager;
+	use Search\Model\Filter\DateBetween;
+
+	class DateBetweenTest extends TestCase {
+
+
+		public $fixtures = [
+			'plugin.Search.Articles'
+		];
+
+		public function testProcess() {
```

Comment:
can you make a test without `from` and `to` - and one where `to` are the only condition, and `last` the same